lib/support/preURL.js

```diff
@@ -0,0 +1,13 @@
+'use strict';
+
+const Promise = require('bluebird'),
+  log = require('intel');
+
+module.exports = {
+  run(context) {
+      return context.runWithDriver((driver) => {
+          log.info('Accessing preURL %s', context.options.preURL);
+          return driver.get(context.options.preURL);
+        }).then(() => Promise.delay(1500));
```

Comment:
aha, right. So basically it's loadAndWait. Hmm, maybe we should expose the SeleniumRunner to pre/post scripts as well, when you don't need the low level access to the raw selenium WebDriver. So content.runWithSeleniumRunner((runner => runner.loadAndWait(context.options.preURL)) instead.